docs/duration.md

```diff
@@ -306,7 +327,7 @@ one.minus(two, { disambiguation: 'balance' });  // => PT2H59M30S
 // Example of not balancing:
 threeYears = Temporal.Duration.from({ years: 3 });
 oneAndAHalfYear = Temporal.Duration.from({ years: 1, months: 6 });
-threeYears.minus(oneAndAHalfYear)  // throws; months are negative and cannot be balanced
+threeYears.minus(oneAndAHalfYear)  // throws; signs are mixed and cannot be balanced
```

Comment:
Isn't the reason why this throws is because months and years are involved?  Seems like the sign has nothing to do with it.  Need to clarify comment.

Reply:
How about "mixed months and years signs cannot be balanced"? It throws because there are mixed signs after balancing, which happens because months and years can't be balanced.